app/models/user.rb

```diff
@@ -1,5 +1,13 @@
 class User < ApplicationRecord
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: Relationship.name,
+    foreign_key: "follower_id",
+    dependent: :destroy
+  has_many :passive_relationships, class_name: Relationship.name,
+    foreign_key: "followed_id",
+    dependent: :destroy
+  has_many :following, through: :active_relationships, source: :followed
```

Comment:
has_many :through để set up quan hệ many-to-many giữa 2 đối tượng trong Rails thông qua 1 bên thứ ba, mặc định tìm đến khoá ngoại theo convention.